container_workflow_tool/utility.py

```diff
@@ -62,6 +64,10 @@ def _get_hostname_url(config):
     return git_url
 
 
+def save_output(logger, output):
```

Comment:
Not sure why we need this helper. Could we not just use logger.info?

Reply:
I will try it and test it by next rebuild iteration.